mne/io/ctf/ctf.py

```diff
@@ -129,6 +129,7 @@ def __init__(self, directory, system_clock='truncate', preload=False,
             if sample_info['n_samp'] == 0:
                 break
             if len(fnames) == 0:
+                # XXX: Buffer size different to c version
                 info['buffer_size_sec'] = \
                     sample_info['block_size'] / info['sfreq']
```

Comment:
This sets what the block size will be when writing to disk, so it shouldn't make a big difference either way. There was a reason I did it this way, but I don't remember what it was offhand. Maybe we actually use it while reading (which wouldn't be great)? Anyway I wouldn't bother changing it.